packages/sidesail/lib/rpc/rpc.dart

```diff
@@ -49,7 +49,10 @@ abstract class RPCConnection extends ChangeNotifier {
       if (!initializingBinary) {
         String? msg = error.toString();
 
-        if (msg.contains('Connection refused')) {
+        if (connectionError != null && connectionError!.contains('Waiting on L1 initial block download to complete')) {
```

Comment:
Triple repeat of long string = 🚩🚩🚩

const pls

Reply:
no